src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -129,10 +146,42 @@ export const NewProposal = ({ navigation, data, query }) => {
         tagList: newProposalData?.tagList,
         termsOfService: hasAcceptedTermsOfService,
         videoUrl: newProposalData?.videoUrl
+
+        // TODO: image and document upload needed here? and if yes, how to do it?
+        //  documentsAttributes: [
+        // 	  {
+        // 		  title: 'sample.pdf',
+        // 		  cachedAttachment:
+        // 		  	'/Users/ardasenturk/Development/SVA/consul-bb/public/system/documents/cached_attachments/user/49/original/f2245afb48be5f906474ad929aacb7f91f408a23.pdf'
+        // 	  }
+        //  ],
       }
     };
 
-    if (!hasAcceptedTermsOfService) return showRegistrationFailAlert();
+    if (!hasAcceptedTermsOfService) return showPrivacyCheckRequireAlert();
+
+    if (newProposalData.image) {
+      const imageUploadData = await uploadAttachment(newProposalData.image, 'image', 'Proposal');
```

Comment:
can `item.name` (with `name: 'image'`) from the fields config be used here instead of the hardcoded `'image'`?

Reply:
alright, let's keep it simple here.